src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlValueConverter.cs

```diff
@@ -588,7 +588,7 @@ protected static byte[] StringToHexBinary(string value)
         {
             try
             {
-                return XmlConvert.FromBinHexString(XmlConvert.TrimString(value), false);
```

Comment:
could TrimString be changed to return ROS?

Reply:
There are benefits to not using a helper method here, see https://github.com/dotnet/runtime/pull/74955#discussion_r961060685

Reply:
fair